charts/workflows/charts/move2kube/values.yaml

```diff
@@ -0,0 +1,15 @@
+sshSecretName: sshkeys # name of the secret holding the ssh keys that will be used by move2kube resources
+brokerName: default
```

Comment:
Pls provide description to the fields. they will be used to render the README.md file when generated by tools.

Reply:
I stole what you did and I put it there, it's mine now :)

Reply:
no problem with that, but generating the values.schema.json and README.md only requires an updated values.yaml with properties for the fields.

This enable them to show a form view in the developer-view of OCP for installing the chart.
BTW, don't forget to install the new icon as well :-p 

![Selection_072](https://github.com/parodos-dev/serverless-workflows-helm/assets/316242/3e28186c-d0bd-43a3-9ea5-8ed44ffaf37c)

Reply:
I commented on the correct place to update. marking this conversation as resolved.